registry/test/registry.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package test
+
+const REGISTRY = "registry"
+const REGISTRY_FULL = "docker.io/library/registry"
+
+type Registry interface {
+	StartRegistry()
+	StopRegistry()
+}
+
+type Docker struct{}
```

Comment:
Not sure if it make sense, why having a interface with empty struct?

Reply:
The functions on Docker and Podman use the same signatures but the difference is the struct on which are applied, this is the reason of this empty structs